app/assets/stylesheets/components/_attachment-meta.scss

```diff
@@ -1,31 +1,19 @@
-.app-c-attachment-meta {
+.gem-c-attachment-meta {
```

Comment:
Minor: we should add a comment to this file to explain why we're using `gem`, as it's inconsistent with all our other 'custom' styles. Sorry I didn't spot this before 😬.

Reply:
I've pushed up one last commit which renames to `app-c` - could you take a look?

Reply:
Looks good to me 👍. 

(I suspect the build failures are because you've also renamed `app/gem-c-attachment`.)